### ISSUE TITLE
Invalid <url-pattern> in servlet mapping when Tomcat 7.0.27 or lower

### ISSUE THREAD

**Issue by HiromuHota** [HiromuHota/pentaho-kettle#25]
When deployed to Tomcat 7.0.27 or lower (e.g., Pentaho BI server 5.4), the following error will happen.
```
SEVERE: Parse error in application web.xml file at jndi:/localhost/spoon/WEB-INF/web.xml
org.xml.sax.SAXParseException; systemId: jndi:/localhost/spoon/WEB-INF/web.xml; lineNumber: 24; columnNumber: 21; Error at (24, 21) : Invalid <url-pattern>  in servlet mapping
```

**Comment by HiromuHota** [comment]
This issue was fixed in 7.0.28. See [here](https://bz.apache.org/bugzilla/show_bug.cgi?id=53356) for details.

**Comment by HiromuHota** [comment]
Resolved by 34c69cb62b74aed7893c63d3fb7c6f130f773e8b.
With this patch, webSpoon can run on 7.0.X and 6.0.X, but not on 5.0.X (BI-server 3.5 to 3.6) (see [here](http://wiki.pentaho.com/display/PEOpen/Pentaho+Release+Product+Version+Matrix) for the detailed version matrix between BI-server and Tomcat).
Currently the version of JSP (for "welcome page"), depended by webSpoon, does not run on 6.0.X.
